src/_locales/en/messages.json

```
@@ -196,6 +196,18 @@
     "message": "Some settings are managed by your system administrator and cannot be changed.",
     "description": "The message, which appears, when settings are pre-defined (as managed options) by administrators."
   },
+  "titleGeneral": {
+    "message": "Add On",
```

```suggestion
    "message": "Add-on",
```
https://design.firefox.com/photon/copy/word-list.html#add-on